### PR TITLE
fix travis-ci

### PR DESCRIPTION
**Pull request by ocefpaf**
Ping @xylar. Alternative for https://github.com/NOAA-ORR-ERD/gridded/pull/39

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,35 +1,37 @@
-language: python
+language: minimal
```
Comment (ocefpaf) [inline .travis.yml]: There is no need to use Travis-CI Python image when using conda.

File: .travis.yml [hunk 2]
```diff
@@ -1,35 +1,37 @@
...
-    - conda install -y --file conda_requirements_test.txt
-    - pip install pytest # for some reason, the conda pytest on linux is broken ?!?
+  - |
+    wget http://bit.ly/miniconda -O miniconda.sh
+    bash miniconda.sh -b -p $HOME/miniconda
+    export PATH="$HOME/miniconda/bin:$PATH"
+    conda config --set always_yes yes --set changeps1 no --set show_channel_urls true
+    conda update --quiet conda
+    conda config --add channels conda-forge --force
+    conda config --set channel_priority strict
```
Comment (ocefpaf) [inline .travis.yml]: This part is what "fixed" the CIs and `strict` will be default in conda 4.7 and this line can be dropped.

File: .travis.yml [hunk 3]
```diff
@@ -1,35 +1,37 @@
...
 before_install:
-    - wget http://bit.ly/miniconda -O miniconda.sh
-    - bash miniconda.sh -b -p $HOME/miniconda
-    - export PATH="$HOME/miniconda/bin:$PATH"
-    - conda config --add channels conda-forge
-    - travis_retry conda create --yes -n TEST python=$TRAVIS_PYTHON_VERSION
-    - source activate TEST
-    - which python
-    - conda install -y --file conda_requirements_test.txt
-    - pip install pytest # for some reason, the conda pytest on linux is broken ?!?
```
Comment (ocefpaf) [inline .travis.yml]: `pip` installing things in a `conda` environment can be troublesome. When you find problems please raise them upstream. BTW, things seems to be OK with conda's pytest now.

**Comment by xylar** [comment]
Nice! @ChrisBarker-NOAA, @ocefpaf is far more expert in this stuff than I am, so I defer to him on this one for sure.

**Comment by ChrisBarker-NOAA** [comment]
Thanks Filipe!

**Pull request merged**